pkg/registry/core/node/storage/storage.go

```diff
@@ -167,6 +168,9 @@ func (r *REST) ResourceLocation(ctx context.Context, id string) (*url.URL, http.
 	return node.ResourceLocation(r, r.connection, r.proxyTransport, ctx, id)
 }
 
+// Implement ShortNamesProvider
+var _ rest.ShortNamesProvider = &REST{}
+
```

Comment:
probably don't need the type assertion here if we're relying on the generic impl